--- a/util/ulockmgr_server.c	2019-01-04 16:33:33.000000000 +0300
+++ b/util/ulockmgr_server.c	2021-08-23 22:31:04.592867579 +0300
@@ -124,7 +124,7 @@
 	return res;
 }
 
-static int closefrom(int minfd)
+static int _closefrom(int minfd)
 {
 	DIR *dir = opendir("/proc/self/fd");
 	if (dir) {
@@ -384,7 +384,7 @@
 		dup2(nullfd, 1);
 	}
 	close(3);
-	closefrom(5);
+	_closefrom(5);
 	while (1) {
 		char c;
 		int sock;
